src/components/IssueType.jsx

```diff
@@ -47,6 +47,12 @@ const IssueType = ({
             onSelect={handleAddressSelect}
           />
           <i
+            style={{
```

Comment:
Would we want to assign this a class and bake in to the components scss library?

Reply:
@theilman7 I think this was me, ultimately what we need to do is add some the ability to place a button on an input. There is a user story for this somewhere on the backlog, so I think this is ok for now.

Reply:
@theilman7 generally if we have css in one of these projects the question is, why aren't these styles in the components project.

Usually that means hey can we move them over, are we missing a component, or maybe we aren't using the right component.

Reply:
@martypowell Sounds good. So essentially, we would come back to this once the backlog is addressed, correct?

Reply:
For now, this should be fine then.

Reply:
yes @theilman7 i'm not sure the best way to manage that in github, but that would be the plan.